public/sass/components/_popper.scss

```diff
@@ -11,50 +11,72 @@
   height: 0;
   border-style: solid;
   position: absolute;
-  margin: 5px;
+  margin: 0px;
 }
 
 .popper .popper__arrow {
   border-color: $tooltipBackground;
 }
 
-.popper[data-placement^='top'] {
-  margin-bottom: 5px;
+.popper__background {
+  background: $tooltipBackground;
+  border-radius: 3px;
```

Comment:
we can use the border radius variable for this 🙂

Reply:
💥 You are 100% right! BRB